test/k8sT/CT.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/gomega"
+
+	. "github.com/cilium/cilium/test/ginkgo-ext"
+	"github.com/cilium/cilium/test/helpers"
+)
+
+var _ = Describe("K8sConnectionTracking", func() {
+	var (
+		kubectl        *helpers.Kubectl
+		ciliumFilename string
+	)
+
+	BeforeAll(func() {
+		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		ciliumFilename = helpers.TimestampFilename("cilium.yaml")
+		DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+			"bpf.monitorAggregation": "none",
```

Comment:
Do you need `monitorAggregation: none` to validate this functionality? Surely the "new" "RST" flag should always trigger a monitor notification, even if monitor aggregation is set to medium (default) ?

Reply:
nope, it's a leftover. I'll remove it.